<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reservationsDialog</class>
 <widget class="QWidget" name="reservationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reservations</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt;&quot;&gt;The next sales are reserved. Please &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt; font-weight:600;&quot;&gt;select one to finish the sale&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt;&quot;&gt; and press the &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt; font-weight:600; font-style:italic;&quot;&gt;Ok&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt;&quot;&gt; button; or press the &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt; font-weight:600; font-style:italic;&quot;&gt;Cancel Selected&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt;&quot;&gt;  button to &lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt; font-weight:600;&quot;&gt;cancel the selected reservation&lt;/span&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:8pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableWidget">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelReservation">
       <property name="text">
        <string>Cancel Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="title">
      <string>Contents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="contentTable">
        <column>
         <property name="text">
          <string>Qty</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="confirmFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>135</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="msgPanel">
        <property name="text">
         <string>Are you sure to cancel the selected reservation?</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chReimbursement">
        <property name="text">
         <string>Make a Reimbursement</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="chNoReimbursement">
        <property name="text">
         <string>DO NOT Make a Reimbursement</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnYes">
          <property name="text">
           <string>Yes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNo">
          <property name="text">
           <string>No</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
